aps/utils/methods.py

```diff
@@ -340,6 +342,12 @@ def calc_average(cell_index_defined: List[int], values: List[float]) -> float:
     Output:
             average  - average value
     """
+    if debug_level >= Debug.VERY_VERBOSE:
+        average, number_of_elements = np.average(
+            values[cell_index_defined], returned=True
+        )
+        print(f'--- Average: {average}  Number of elements: {number_of_elements}')
+        return average
```

Comment:
Even though this actually now is a different return because of the `debug_level`, I recognize that currently it does the same thing, just with some more stuff. 

Is this a huge array? If not, I don't think there's much of a cost to doing this `returned=True` option anyways, so maybe we can just do that anyways, but only use it if the `debug_level` is high enough?

Otherwise this is fine as is.

Reply:
I change this to always use returned=True in this average calculation.